docs/schema/cookieauthdata-element-sps15xsdsearchset2.md

```diff
@@ -1,26 +1,26 @@
 ---
 title: "CookieAuthData element (SPS15XSDSearchSet2)"
+description: Outlines information, a definition, and elements and attributes for the CookieAuthData element in Sharepoint.
 manager: arnek
-ms.date: 3/9/2015
+ms.date: 06/09/2022
 ms.audience: ITPro
 ms.topic: article
 ms.prod: sharepoint
 ms.localizationpriority: medium
 ms.assetid: 7a37cb4e-e375-87c6-3560-92a9f679b297
-description: "Last modified: March 09, 2015"
 ---
 
 # CookieAuthData element (SPS15XSDSearchSet2)
 
  
   
 ## Element information
+**Element type**: tns:CookieAuthData
```

Comment:
We've been directed to keep Markdown tables as tables. You can use "Information/Location" or "Name/Value" as headers, or if you find a more specific header that fits, use that.